### ISSUE TITLE
Using Presets in multiple rigs

### ISSUE THREAD

**Issue by PomboPombudo** [Mystfit/IK-FK-Snapping-for-Blender#1]
Hi! This is not really a bug, but something that is being problematic in my actual project. I'm working with multiple rigs I'd really like to have a system that let me separete the saved presets by each rig, right now I have to see all the presets for all my rigs at all times and it's a little tricky.

I'm not into script myself, for that reason really apreciate this addon, thanks a lot!

**Comment by Mystfit** [comment]
Thanks, that's a really good suggestion actually. How about a checkbox to filter the presets based on the active armature?

**Comment by PomboPombudo** [comment]
yes, I think it's a nice solution to the problem. Also I was thinking, becouse I always use custom properties on my rigs, to be able to control the snaps by drivers. However, I don't know how difficult is to do that.

**Comment by Mystfit** [comment]
My addon registers two operators, SnapIKToFKOperator and SnapFKToIKOperator. You can call these operators as long as the following attributes are present:
```
bpy.context.scene.armature_name,
bpy.context.scene.FK_control_upper_name,
bpy.context.scene.FK_control_lower_name,
bpy.context.scene.FK_control_end_name,
bpy.context.scene.IK_control_upper_name,
bpy.context.scene.IK_control_lower_name,
bpy.context.scene.IK_control_end_name,
bpy.context.scene.IK_control_name,
bpy.context.scene.IK_control_pole_name
```

**Comment by Mystfit** [comment]
> Hi! This is not really a bug, but something that is being problematic in my actual project. I'm working with multiple rigs I'd really like to have a system that let me separete the saved presets by each rig, right now I have to see all the presets for all my rigs at all times and it's a little tricky.
> 
> I'm not into script myself, for that reason really apreciate this addon, thanks a lot!

For the meantime, you could prefix your preset with the name of the armature to keep the presets seperate from each other in the preset menu. It might be cluttered at this stage but you'll be able to keep different presets.

**Comment by PomboPombudo** [comment]
I'll do as you say. Thanks!!
